app/helpers/application_helper.rb

```diff
@@ -54,6 +54,32 @@ def tail(collection)
     collection.to_a[head_size..-1].to_a
   end
 
```

Comment:
These are fine with me, but let's put them in their own class. ./lib/sparklines perhaps? They seem like static methods, so they can probably be defined as class methods on a `module Sparklines`. Side question: what does this word "sparklines" mean?

Reply:
done, also moved more stuff to problems model.